microcosm_flask/tests/test_metrics.py

```diff
@@ -120,7 +118,6 @@ def foo():
             tags=[
                 "endpoint:foo.search.v1",
                 "backend_type:microcosm_flask",
-                "classifier:404",
```

Comment:
Unrelated, but the tests were being skipped by default, and wouldn't pass when enabled.